src/javascript/app/base/socket_general.js

```diff
@@ -162,9 +161,6 @@ const BinarySocketGeneral = (() => {
                 }
                 break;
             }
-            case 'RateLimit':
```

Comment:
Are you removing the whole functionality of the RateLimit alert or is it appearing somewhere else?
Don't we need this functionality anymore?

Reply:
@mahdiaryayi-fs rate limit alert for mt5 seems to be appearing somewhere else

Reply:
@fiona-deriv then we should remove that other one and keep this. this is used for our whole website.

Reply:
okay thanks, noted